upi/azure/azuredeploy.json

```diff
@@ -0,0 +1,1072 @@
+{
+  "$schema" : "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion" : "1.0.0.0",
+  "parameters" : {
+    "image" : {
+      "type" : "string",
+      "metadata" : {
+        "description" : "OS to use.Red Hat Enterprise Linux"
+      }
+    },
+    "BootstrapIgnition" : {
```

Comment:
looks like ARM has securedata, we probably should do that for bootstrap ignition.